websites/C/Crunchyroll/presence.ts

```diff
@@ -109,14 +109,31 @@ presence.on("UpdateData", async () => {
   }
 
   if (iFrameVideo !== false && !isNaN(duration)) {
-    const videoTitle = document.querySelector(".ellipsis .text-link span"),
-      episod = document.querySelectorAll("#showmedia_about_media h4"),
-      epName = document.querySelector("h4#showmedia_about_name"),
-      episode = episod[1].innerHTML + " - " + epName.innerHTML,
-      timestamps = presence.getTimestamps(
-        Math.floor(currentTime),
-        Math.floor(duration)
-      );
+    let videoTitle,
+    type,
+    episode,
+    season;
+    if (document.location.hostname.startsWith("beta")) {
+      const episodeid = document.location.pathname.match(/watch\/(.*)\/.*/),
+      url = `https://beta-api.crunchyroll.com/cms/v2/US/M3/-/objects/${episodeid[1]}?Signature=A58Y6P76n7z72bcueov582lZiJcWCoJ-~AKn4y6khza30izYCquaIvRH9lhX6mY2goJpCyc9c74mI4QdWwYna8kfHUMiZmgD3ofvSGyZX88XTiR~nSVfNmBmRiGpPAR1MkRYe3pFWbAHOqPMNB89EKCg1Ho~LdecTbuQ3seOMip9QYYH6Z8T0GCFJlOxS3sMKT0hPak~nYHrj4La-IEmAe6PwbSxg59j3iIB7u8ft020CIkel68O~PjPL~MSot1eU7GZMGVQAkAOnKoZFLt~H2KeFQdHzsfp8Ns39jKNcvMwQ9BGU8XlEWQTBCNrWOycnTvafvBCPqFYXWMiTtiNXg__&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9iZXRhLWFwaS5jcnVuY2h5cm9sbC5jb20vY21zL3Y~L1VTL00zLy0vKiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTYyMzA5MDU0M319fV19&Key-Pair-Id=APKAJMWSQ5S7ZB3MF5VA`,
+      obj = await (await fetch(url)).json(),
+      metadata = await obj.items[0],
```

Comment:
I hope that by this ppl will not hit ratelimit or something 😂

Reply:
whelp it did :). I've decided I'll just settle with giving the title of the season rather than having to mess with the api just to get the season number.

Reply:
🤔 couldn't you like do the object with this stuff and just check if it is the same for example name of video and just if its not download data again?

Reply:
Since I'm not generating a token here, it'll eventually expire and I'd have to supply a link again. Figuring out how to generate one is kind of a pain.

Reply:
got it.